stack-9.8.2.yaml

```diff
@@ -1 +0,0 @@
-resolver: nightly-2024-04-05
```

Comment:
While you're at it, you could also add 9.12.2 which is the current nighly :smile_cat:

Reply:
There's a separate PR for supporting GHC 9.12. Had to bump base for that.